components/list/demo/list-nested.html

```diff
@@ -224,6 +226,66 @@ <h2>Nested Custom Items</h2>
 				</template>
 			</d2l-demo-snippet>
 
+			<h2>Side nav item</h2>
+
+			<d2l-demo-snippet>
+				<template>
+					<d2l-list grid style="width: 334px;">
+						<d2l-list-item-nav-button key="L1-1" label="Welcome!" color="#006fbf" expandable expanded draggable>
+							<d2l-list-item-content>
+								<div>Welcome!</div>
+							</d2l-list-item-content>
+							<d2l-list slot="nested" grid>
+								<d2l-list-item-nav-button key="L2-1" label="Syallabus Confirmation" draggable>
+									<d2l-list-item-content>
+										<div><d2l-icon style="margin-right: 0.7rem;" icon="tier2:file-document"></d2l-icon>Syallabus Confirmation</div>
+										<div slot="secondary"><d2l-tooltip-help text="Due: May 2, 2023 at 2 pm">Due: May 2, 2023</d2l-tooltip-help></div>
+									</d2l-list-item-content>
+								</d2l-list-item-nav-button>
+							</d2l-list>
+						</d2l-list-item-nav-button>
+						<d2l-list-item-nav-button key="L2-2" label="Unit 1: Poetry" color="#29a6ff" expandable expanded draggable>
+							<d2l-list-item-content>
+								<div>Unit 1: Fiction</div>
+								<div slot="secondary"><d2l-tooltip-help text="Starts: May 2, 2023 at 2 pm">Starts: May 2, 2023</d2l-tooltip-help></div>
+							</d2l-list-item-content>
+							<d2l-list slot="nested" grid>
+								<d2l-list-item-nav-button key="L3-2" label="Fiction" draggable>
+									<d2l-list-item-content>
+										<div><d2l-icon style="margin-right: 0.7rem;" icon="tier2:file-document"></d2l-icon>Fiction</div>
+									</d2l-list-item-content>
+								</d2l-list-item-nav-button>
+								<d2l-list-item-nav-button key="L3-2" label="Ten rules for writing fiction" draggable>
+									<d2l-list-item-content>
+										<div><d2l-icon style="margin-right: 0.7rem;" icon="tier2:file-document"></d2l-icon>Ten rules for writing fiction</div>
+									</d2l-list-item-content>
+								</d2l-list-item-nav-button>
+							</d2l-list>
+						</d2l-list-item-nav-button>
+					</d2l-list>
+				</template>
+				<script type="module">
```

Comment:
I'm guessing this `script` element is outside of the `template` since we don't want to show it in the demo page, ya? If the `script` element is placed inside the `template` then we don't have timing issues when trying to wire up handlers on page load, avoiding the need for the rAF, but I know sometimes we don't want to show some of this script.

Aside from the timing, of we're ever worried about conflicts between demos, it's possible to scope things with something like this:
```html
<script>
		(demo => {
			const elem = demo.querySelector(...);
			...
		})(document.currentScript.parentNode);
</script>
```

Reply:
Ah nope - I did want it shown on the demo page. I've moved it and updated to scope it. Thanks!